cdap-ui/app/features/services/controllers/tabs/runs/tabs/status/make-request-ctrl.js

```diff
@@ -108,8 +110,12 @@ angular.module(PKG.name + '.feature.services')
       }
 
       dataSrc.request(requestObj)
-        .then(function(res) {
+        .then(function success(res) {
           vm.response = res;
+          vm.loading = false;
```

Comment:
So we should be able to show some standard response when the POST is successful. Right now backend just sends 200 and we don't know if something actually happened and is successful or now.